src/components/orgunits/OrgUnitFieldSelect.js

```diff
@@ -34,9 +34,9 @@ const OrgUnitFieldSelect = () => {
     const attribute = attributes.find((a) => a.id === orgUnitField)
 
     return (
-        <>
+        <div className={styles.orgUnitField}>
             <SelectField
-                label={i18n.t('Use associated geometry')}
+                prefix={i18n.t('Use associated geometry')}
```

Comment:
Suggest prefix:

`Associated geometry:`

Reply:
Following the sketch from Joe: 

![image](https://user-images.githubusercontent.com/548708/220334573-2d84bbd5-4f40-468a-a5b1-1609f127cca8.png)